doc/architecture/decisions/0002-use-express-web-framework.md

```diff
@@ -0,0 +1,41 @@
+# 2. Use Express web framework
+
+Date: 2017-06-09
+
+## Status
+
+Accepted
+
+## Context
+
+A web framework is needed in order to provide a level of abstraction over low
+level language features. Effort spent solving non-domain specific problems
+should be kept to an absolute minimum.
+
+## Decision
+
+We will use the [Express](http://expressjs.com/) web framework. This will
+provide a lot of well
+[documented](http://expressjs.com/en/resources/books-blogs.html) features and
+functionality. Whilst providing a fast, unopinionated, minimalist web
+framework.
+Express is one of the most popular web frameworks available for NodeJS and as
+such has been well tested and is deployed across many
+[organisations](http://expressjs.com/en/resources/companies-using-express.html).
+
+## Consequences
+
+Due to Express being a minimalist web framework, when additional features are
+required they are added as
+[middleware](http://expressjs.com/en/resources/middleware.html) modules.
+A working knowledge of existing modules will help with reducing duplication of
+effort by where modules may already exist within the community.
```

Comment:
This sentence `A working knowledge of existing modules will help with reducing duplication of effort by where modules may already exist within the community` doesn't scan too well.
Could it be shortened to something like `Duplication of effort for common tasks may be reduced by using modules that already exist within the community`